### ISSUE TITLE
Create wireframes for in UI create/edit of DOI

### ISSUE THREAD

**Issue by rickjohnson** [samvera-labs/doi_registrar#17]
relates to #8

**Comment by jamesvanmil** [comment]
Here's a screenshot of what we implemented in our Hyrax 1.0.5 app.

It's essentially the same form we developed for Curate, with some light modification to help information.

<img width="1191" alt="screen shot 2018-03-06 at 8 52 47 am" src="https://user-images.githubusercontent.com/2651187/37035856-dd4375de-211b-11e8-9044-a2fd45bbb3d7.png">

**Comment by rickjohnson** [comment]
Some institutions may want to limit DOI creation to administrators.  So, UI may be something configurable (or we at least need to decide on a default case).

**Comment by hjsyoo** [comment]
@rickjohnson Agreed, having an admin-only option could be useful. Also, would recommend using "persistent" over "permanent" when referring to DOIs. The latter may give the impression that many aspects of DOIs are unchangeable?
